meetings/2023-06-08.md

```diff
@@ -3,18 +3,34 @@
 ## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
 
 ### Attending
-- 
+- Kris becker
+- Stuart Robbins
+- Kelvin Rodriguez
+- Adam Paquette
+- Victor Silva
+- Trent Hare
+- Ross Beyer
+
 ## Agenda / Notes
 - Revisit - SpiceQL.
+  - Potential replacement for the SPICE server
   - Significant performance improvement discussion (sub-minute right now, target is 5 seconds.)
-- File suffixes, lbl, pvl, txt, oh my.
+  - ASC is working on this.
+    - Caching is in / going in.
+    - Current time is 60 seconds (for a CTX image).
+    - Plans to remove the other slowdowns.
+    - These timings are only for the remote server, not local ISIS/ALE that has the files locally.
+    - This is neither integrated nor released.
+- File suffixes: lbl, pvl, txt, etc.
+  - Deferred
+- LTS Discussion.
 
 ## Discussions for next meeting
-
-- 
+- M1/M2 folks update. 
+- LTS update (it likely will have shipped unless more bugs are identified and fixed)
```

Comment:
Could we get the file suffixes add to this list due to the deferred status

Reply:
It's in line 11 of the agenda for next month. Re-add here too?

Reply:
Ah, missed that! No I think we only need the one